pkg/auth/handler/webapp/alternatives.go

```diff
@@ -103,7 +103,7 @@ func handleAlternativeSteps(ctrl *Controller) {
 			}
 			switch node.GetCreateAuthenticatorStage() {
 			case authn.AuthenticationStagePrimary:
-				panic("Magic link as primary authenticator is not yet supported")
+				choiceStep = webapp.SessionStepCreateAuthenticator
```

Comment:
Both primary and secondary stage are having the same `choiceStep`, don't need the switch case?

Reply:
Updated